config-linux.md

```diff
@@ -359,7 +359,12 @@ For more information, see the kernel cgroups documentation about [cpusets][cgrou
 The following parameters can be specified to set up the controller:
 
 * **`shares`** *(uint64, OPTIONAL)* - specifies a relative share of CPU time available to the tasks in a cgroup
-* **`quota`** *(int64, OPTIONAL)* - specifies the total amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+* **`quota`** *(int64, OPTIONAL)* - specifies the replenished amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+    A value of `-1` indicates that the cgroup does not have any bandwidth restriction in place (i.e., an unconstrained bandwidth group).
+    Any (valid) positive value(s) no smaller than `burst` will enact the specified bandwidth limit.
+* **`burst`** *(uint64, OPTIONAL)* - specifies the maximum amount of accumulated time in microseconds for which all tasks in a cgroup can run additionally for burst during one period (as defined by **`period`** below)
```

Comment:
```suggestion
* **`burst`** *(uint64, OPTIONAL)* - specifies the maximum amount of accumulated time in microseconds for which all tasks in a cgroup can run additionally for burst during one period (as defined by **`period`** below). This feature just be supported in the Linux 5.14 or newer
```

Reply:
Normally we do not highlight the kernel version requirement in runtime spec.

Reply:
OK, Got it

Reply:
Personally, I think it would be valuable for us to list the relevant kernel version that added each feature (above some global minimum we assume, which I don't think is currently codified anywhere in the spec even though we definitely do assume _some_ minimum), but I agree that it doesn't make sense to do it just for this one instance. :smile: